packages/vertexai/src/types/enums.ts

```diff
@@ -240,3 +240,29 @@ export enum Modality {
    */
   DOCUMENT = 'DOCUMENT'
 }
+
+/**
+ * Generation modalities to be returned in generation responses.
+ *
+ * @beta
+ */
+export const ResponseModality = {
```

Comment:
This is the code object we agreed we should be exporting instead of TS enums so I get that, but we've had build issues in the past mixing JS code in types files so we should probably put these in a separate file. Looks like it's not causing build issues now so maybe we can move it along with the others whenever we plan to convert all our enums to JS objects.